server/migrations/20180228152639-create-user-assignees.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('userAssignees', {
+      userId: {
+        type: Sequelize.STRING,
+        onDelete: "CASCADE",
+        onUpdate: "CASCADE",
+        references: {
+          model: "Users",
+          key: "id",
+          as: "userId"
+        }
+      },
+      assigneeId: {
+        type: Sequelize.STRING(30),
+        onDelete: "CASCADE",
+        onUpdate: "CASCADE",
+        references: {
+          model: "Assignees",
```

Comment:
Instead of having an assignees table, why can't this reference the users table. Because an assignee is a user. @Mnickii suggestions are welcomed. So a user as assignee may belongToMany incidents and an incident can belong to several assignees.

Reply:
I thought when we were thinking of join table for users and assignees 🤕 Since we can't have two join tables for users and incidents i.e `witnesses belongToMany incidents and incidents can belong to many witnesses as well`, we keep the witnesses model as it is in develop? @kimobrian

Reply:
Updating this PR for the JOIN table

Reply:
Let me look at it. Will get back to you.